locales/templates/app.pot

```diff
@@ -579,3 +598,42 @@ msgctxt "onboarding_search_content_feedback"
 msgid "Thanks for your feedback!"
 msgstr ""
 
+#. String displayed with an illustration to introduce a new feature
+msgctxt "onboarding_ocr_title_more"
+msgid "Do more with screenshots"
+msgstr ""
+
+#. String displayed to give examples of what the new feature can do
+msgctxt "onboarding_ocr_content_examples"
+msgid "Look up products, copy coupon codes, and share quotes in your screenshots"
+msgstr ""
+
+#. In-app promote dialog
+msgctxt "dialogue_feedback_title"
+msgid "Love Firefox ScreenshotGo?"
```

Comment:
%1$s has been used for "Firefox ScreenshotGo", probably better to be consistent with that